README.md

```diff
@@ -926,7 +926,13 @@ The actual stdout of the downstream `am` command.
 
 ### mobile: getContexts
 
-Retrieves a webviews mapping based on CDP endpoints
+Retrieves a WebViews mapping based on CDP endpoints
+
+#### Arguments
+
+Name | Type | Required | Description | Example
+--- | --- | --- | --- | ---
+waitForWebviewMs | number | no | Tells UiAutomator2 driver for how long (in milliseconds) to wait for web view(s) to appear since UiAutomator2 driver v2.53.0. Chrome process running on the device under test could fail to create `chrome_devtools_remote`, then the chromedriver raised `failed to connect to socket 'localabstract:chrome_devtools_remote'` error to UiAutomator2 driver. It could cause no WebViews found result, althought a couple of retrial succeeds in finding WebView(s). This argument helps to keep trying to get WebView(s) up to the given time milliseconds as one command call. This issue tends to occur Chrome v115 and over so far. [issues#19251](https://github.com/appium/appium/issues/19251) has more detail logs. `0`ms by default, then UiAutomator2 driver checks the WebView(s) availability once. | 10000
```

Comment:
... contains more details

Reply:
If set to `0` (the default value), then UiAutomator2 driver only checks the WebView(s) availability once

Reply:
If a Chrome process running on the device under test fails to create a connection to devtools socket, then the chromedriver raises an error similar to `failed to connect to socket 'localabstract:chrome_devtools_remote'` in UiAutomator2 driver -> the time in both parts of the sentence must be coordinated

Reply:
It could cause no WebViews found result, althought a couple of retrials may fix it.